lib/ld-eventsource/client.rb

```diff
@@ -268,14 +268,14 @@ def connect
             headers: build_headers
           })
           if cxn.status.code == 200
-            content_type = cxn.headers["content-type"]
+            content_type = cxn.content_type.mime_type
```

Comment:
`cxn.content_type` returns a struct, `mime_type` gets us the appropriate string which matches the header.